h2/src/test/org/h2/test/unit/TestDateTimeUtils.java

```diff
@@ -22,6 +27,12 @@
      * @param a ignored
      */
     public static void main(String... a) throws Exception {
+        if (a.length == 1) {
+            if ("testUtc2Value".equals(a[0])) {
+                new TestDateTimeUtils().testUTC2Value(true);
+                return;
+            }
+        }
```

Comment:
looks like leftover debug code?

Reply:
It's just a way to run test for all timezones manually. It's too slow to run it as a part of usual testing process.

Reply:
I can remove it if this code is not wanted here.

Reply:
no, just add a comment please